### PR TITLE
Bump up `Reuters` parser to version V1_1

### PR DESCRIPTION
**Pull request by MaxDall**
This gets `Reuters` up and running again.

### CODE REVIEW

File: src/fundus/publishers/us/reuters.py [hunk 1]
```diff
@@ -59,3 +61,18 @@ def topics(self) -> List[str]:
             # Remove empty topics and duplicates deterministically
             processed_topics = list(dict.fromkeys(topic for topic in topics if topic))
             return processed_topics
+
+    class V1_1(V1):
```
Comment (addie9800) [inline src/fundus/publishers/us/reuters.py:65]: There are some cases of subheadlines: e.g. https://www.reuters.com/world/us/takeaways-state-union-address-2024-03-08/ or https://www.reuters.com/markets/global-markets-wrapup-1-2024-03-08/

File: src/fundus/publishers/us/reuters.py [hunk 2]
```diff
@@ -59,3 +61,18 @@ def topics(self) -> List[str]:
...
+        VALID_UNTIL = date.today()
+        _paragraph_selector = XPath("(//div[starts-with(@data-testid, 'paragraph')])[position() > 1]")
```
Comment (addie9800) [inline src/fundus/publishers/us/reuters.py:67]: A lot of sports related articles have this tag at the end of the article `--Field Level Media` e.g. https://www.reuters.com/sports/basketball/hot-shooting-suns-wear-down-raptors-2024-03-08/ or https://www.reuters.com/sports/nhl/filip-forsbergs-hat-trick-sends-predators-past-sabres-2024-03-08/

**Review by addie9800 — CHANGES_REQUESTED** [review]
Thanks for updating this 👍

File: src/fundus/publishers/us/reuters.py (hunk 1 above)
Comment (MaxDall) [reply src/fundus/publishers/us/reuters.py:67]: Nice you found this 👍 I added the selector to the original version as well.

**Review by addie9800 — APPROVED** [review]

**Pull request merged**